code-challenges.test.js

```diff
@@ -11,46 +11,176 @@
+
+
+let fibinochi= () => {}
+    const newArray = [0, 1];
+    for(var i = 2; i < newArray; i++) {
+       const a = (i - 1);
+       const b = (i - 1);
+       newArray.push(a + b);
+    }
+
+
+  
+       
+// QUESTION 2) Create a function that takes in an array and returns a new array of only odd numbers sorted from least to greatest.
+
+//PSUEDO CODING
+// create a functionn that takes in a an array
+// Output: new array of only odd numbers sorted from least to greatest.
+// Use the .filter method the filter through the array ang grab all of the odd numbers. Create a another variable for the new array, .sort through that array to get the number from leats to greatest
+// Expected output: -9, 7, 9, 199]
+// Expected output: 7,23, 67,-823]
+
+
 const fullArr1 = [4, 9, 0, "7", 8, true, "hey", 7, 199, -9, false, "hola"]
-// Expected output: [-9, 7, 9, 199]
+// // Expected output: [-9, 7, 9, 199]
 
 const fullArr2 = ["hello", 7, 23, -823, false, 78, null, "67", 6, "number"]
-// Expected output: [-823, 7, 23]
+// Expected output: [7,23, 67,-823]
 
 
-// b) Create the function that makes the test pass.
+// 1a) Create a test with expect statements for each of the variables provided.
 
+// FAIL  ./code-challenges.test.js
+// theOddNumbers
+//   ✕ returns a new array of only odd numbers sorted from least to greatest. (1 ms)
 
-// --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+// ● theOddNumbers › returns a new array of only odd numbers sorted from least to greatest.
+
+//   ReferenceError: theOddNumbers is not defined
+
+describe("theOdds", () => {
+    it("returns a new array of only odd numbers sorted from least to greatest.", () => {
+      expect(theOdds(fullArr1)).toEqual([-9, 7, 9, 199]);
+      expect(theOdds(fullArr2)).toEqual([7,23, 67,-823]);
+    });
+  })
+
+
+
+// 2b) Create the function that makes the test pass.
+
+// TRY One  🤷🏾‍♀️
+const onlyNumbers = fullArr1.filter( 
+    element => typeof element === "number"
+); 
+
+const odds = onlyNumbers.filter(number => {
+    return number % 2 !== 0;
+});
+
+const sorted= odds.sort
+ 
+
+
+// Question 3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
-// a) Create a test with expect statements for each of the variables provided.
 
 const numbersToAdd1 = [2, 4, 45, 9]
-// Expected output: [2, 6, 51, 60]
+// // Expected output: [2, 6, 51, 60]
 
 const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
+// // Expected output: [0, 7, -1, 11]
 
 const numbersToAdd3 = []
-// Expected output: []
+// // Expected output: []
+
+//PSUEDO CODE
+// create a functionn that takes in a an array
+// Input: numbersToAdd1,2,3, which are the original arrays given to us
+// Output: pushing numbers to a  array with the accumulation sum of the original array
```

Comment:
Okay but this pseudo code doesn't express **"how"** to solve this problem-- just what the problem is.